vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -47,19 +47,20 @@ type internal FSharpCheckerProvider
             // When the F# background builder refreshes the background semantic build context for a file,
             // we request Roslyn to reanalyze that individual file.
             checker.BeforeBackgroundFileCheck.Add(fun (fileName, extraProjectInfo) ->  
-               async {
-                try 
-                    match extraProjectInfo with 
-                    | Some (:? Workspace as workspace) -> 
-                        let solution = workspace.CurrentSolution
-                        let documentIds = solution.GetDocumentIdsWithFilePath(fileName)
-                        if not documentIds.IsEmpty then 
-                            analyzerService.Reanalyze(workspace,documentIds=documentIds)
-                    | _ -> ()
-                with ex -> 
-                    Assert.Exception(ex)
+                async {
+                    try 
+                        match extraProjectInfo with 
+                        | Some (:? Workspace as workspace) -> 
+                            let solution = workspace.CurrentSolution
+                            let documentIds = solution.GetDocumentIdsWithFilePath(fileName)
+                            if not documentIds.IsEmpty then 
+                                analyzerService.Reanalyze(workspace, documentIds=documentIds, highPriority=true)
```

Comment:
Also wondering if this is going to get us faster squiggles in the editor. Will probably move this change out of this PR though.